{"name": "test_div[-10.2-0-ZeroDivisionError]", "status": "broken", "statusDetails": {"message": "ZeroDivisionError: float division by zero", "trace": "num1 = -10.2, num2 = 0, expection = <class 'ZeroDivisionError'>\n\n    @pytest.mark.parametrize(\"num1, num2, expection\", {\n        (10, 2, 5),\n        (100000000, 2, 50000000),\n        (0, 10, 0),\n        (10, 3, 3.3333),\n        (5, 4, 1.25),\n        (20, -4, -5),\n        (100000000, -5, -20000000),\n        (0, -10, 0),\n        (20, -6, -3.3333),\n        (18, -8, -2.25),\n        (-30, 10, -3),\n        (-5000000000, 5, -1000000000),\n        (-20, 3, -6.6666),\n        (-30, 8, -3.75),\n        (-30, -10, 3),\n        (-18, -8, 2.25),\n        (-20, -6, 3.3333),\n        (10.2, 2, 5.1),\n        (555555555.5, 5, 111111111.1),\n        (20.2, 3, 6.0666),\n        (10, 0.2, 50),\n        (1800000000, 2.25, 800000000),\n        (10, 3.3, 3.030303),\n        (5.5, 0.5, 11),\n        (20.2, 3.3, 6.121212),\n        (-10.2, 2, -5.1),\n        (-555555555.5, 5, -111111111.1),\n        (-20.2, 3, -6.0666),\n        (10, -0.2, -50),\n        (1800000000, -2.25, -800000000),\n        (10, -3.3, -3.030303),\n        (-5.5, -0.5, 11),\n        (-20.2, -3.3, 6.121212),\n        (10, 0, ZeroDivisionError),\n        (-10, 0, ZeroDivisionError),\n        (10.2, 0, ZeroDivisionError),\n        (-10.2, 0, ZeroDivisionError),\n        (10, \"a\", TypeError),\n        (\"a\", 10, TypeError),\n        (10.5, \"abc\", TypeError),\n        (\"abc\", 10.5, TypeError),\n        (-10, \"a\", TypeError),\n        (\"a\", -10, TypeError),\n        (-10.5, \"abc\", TypeError),\n        (\"abc\", -10.5, TypeError)\n    })\n    def test_div(num1, num2, expection):\n>       assert div(num1, num2) == expection\n\ntest_div.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\na = -10.2, b = 0\n\n    def div(a, b):\n>       return a / b\nE       ZeroDivisionError: float division by zero\n\ndev.py:2: ZeroDivisionError"}, "parameters": [{"name": "num1", "value": "-10.2"}, {"name": "num2", "value": "0"}, {"name": "expection", "value": "<class 'ZeroDivisionError'>"}], "start": 1589597518549, "stop": 1589597518549, "uuid": "ad5f785b-2c67-4aad-8d24-47b9b6dbd2f4", "historyId": "c821ea217fcec6b5631395e4cb3d21cc", "testCaseId": "d61eec9ff444e0577c99edd05c5a5c5c", "fullName": "test_div#test_div", "labels": [{"name": "suite", "value": "test_div"}, {"name": "host", "value": "MS-WTAHPQSUHCOQ"}, {"name": "thread", "value": "8536-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_div"}]}